requirements/edx/github.txt

```diff
@@ -32,7 +32,7 @@ git+https://github.com/hmarr/django-debug-toolbar-mongo.git@b0686a76f1ce3532088c
 -e git+https://github.com/jazkarta/ccx-keys.git@e6b03704b1bb97c1d2f31301ecb4e3a687c536ea#egg=ccx-keys
 
 # Our libraries:
--e git+https://github.com/edx/XBlock.git@e1831fa86bff778ffe1308e00d8ed51b26f7c047#egg=XBlock
+-e git+https://github.com/cpennington/XBlock.git@222dca51d5e69b3ac436c37c897978411bb4364f#egg=XBlock
```

Comment:
N.B. this needs to be changed after https://github.com/edx/XBlock/pull/302 is merged.